awesomemotive/wpforms-multiselect-js
# Add customOpener configuration option and handle clicks

Fixes #3 

Test steps:
- add select element which you want multiselect to be applied for
- add some `<button id="foo">`
- initialize multiselect with option `customOpener: '#foo'`
- click the button and make sure multiselect is open

You can see this in real use on this branch of WPForms:
https://github.com/awesomemotive/wpforms-plugin/pull/8341/commits/12d59d1b75ea3717ae43784bfd913f9df7282016
- go to WPForms > Entries > Select some form with entries
- on the entries list page click gear icon in right top corner of the table
- 
![vokoscreenNG-2023-10-19_12-49-10](https://github.com/awesomemotive/wpforms-multiselect-js/assets/2388617/21f87507-9e83-4dee-a277-2761c0ba8b9f)
